databricks/koalas/tests/test_series.py

```diff
@@ -170,6 +170,65 @@ def test_rename_method(self):
         # s.rename(lambda x: x**2, inplace=True)
         # self.assert_eq(kser, pser)
 
+    def test_rename_axis(self):
+
+        index = pd.Index(["A", "B", "C"], name="index")
+        pser = pd.Series([1.0, 2.0, 3.0], index=index, name="name")
+        kser = ks.from_pandas(pser)
+
+        self.assert_eq(
+            pser.rename_axis("index2").sort_index(), kser.rename_axis("index2").sort_index(),
+        )
```

Comment:
Could you add a test like `(kser + 1).rename_axis("index2").sort_index()`?

Reply:
Yes, `resolved_copy` is now used !

Reply:
Cool, @LucasG0 . But maybe I think we have a bit more better way ?

Could you look at https://github.com/databricks/koalas/pull/1843#discussion_r507027779 ?